### PR TITLE
Improve parameters documentation & fix-up accelerators page

### LINKED ISSUE #27: packet module has a section on MPLS but it's missing from the list of submodules at the top of the document
see summary.

### PR DESCRIPTION
**Pull request by floren**
Also closes #27

### CODE REVIEW

File: configuration/accelerators.md [hunk 1]
```diff
@@ -121,14 +121,15 @@ The fulltext accelerator supports a few options which allow for refining the typ
...
-| -ignoreUUID | Enable a filter to ignore UUID/GUID values.  Some logs will generate a unique UUID for every entry which incurs significant overhead and provides very little value. | `-ignoreUUID` | DISABLED |
-| -ignoreFloat | Disable filters which ignore floating point numbers. Logs where floating point numbers are used for filters can make use of `-accptFloat`. | `-acceptFloat` | DISABLED |
-| -maxInt | Enable a maximum integer value | Enable a filter that will only index integers below a certain size.  This can be valuable when indexing data that such as HTTP access logs.  You want to index the return codes, but not the response times and data sizes. |
-
-NOTE: Make sure you understand your data before enabling the `acceptTS` and `-acceptFloat` flags as these can dramatically bloat the index when using the index engine.  The Bloom engine is less impacted by orthogonal data such as timestamps and floating point numbers.
+| -acceptTS | By default the fulltext accelerator attempts to identify and ignore timestamps in the data. This flag disables that behavior and allows timestamps to be indexed. | `-acceptTS` | DISABLED |
```
Comment (djfritz) [inline configuration/accelerators.md:124]: Is it acceptTS or ignoreTS?

File: configuration/accelerators.md [hunk 2]
```diff
@@ -121,14 +121,15 @@ The fulltext accelerator supports a few options which allow for refining the typ
...
+| -acceptFloat | By default, the fulltext accelerator attempts to identify and ignore floating-point numbers, since they typically vary widely and are not explicitly queried. Setting this flag disables that behavior and allows floating-point numbers to be indexed. | `-acceptFloat` | DISABLED |
```
Comment (djfritz) [inline configuration/accelerators.md:125]: Is it acceptFloat or ignoreFloat?

File: configuration/accelerators.md (hunk 1 above)
Comment (floren) [reply configuration/accelerators.md:124]: Both. -acceptTS means we should index timestamps. Reading the code, by default it ignores timestamps. -ignoreTS is implemented so we can make the decision explicit.

File: configuration/accelerators.md (hunk 2 above)
Comment (floren) [reply configuration/accelerators.md:125]: As with -acceptTS, -acceptFloat and -ignoreFloat both exist. ignoreFloat is default.

File: configuration/accelerators.md (hunk 1 above)
Comment (djfritz) [reply configuration/accelerators.md:124]: Should we put both flags in there? We use `-ignoreTS` in examples.

File: configuration/accelerators.md (hunk 2 above)
Comment (djfritz) [reply configuration/accelerators.md:125]: Same as above comment.

**Comment by floren** [comment]
ptal

**Pull request merged**